mu-plugins/blocks/global-header-footer/postcss/_common.pcss

```
@@ -81,10 +81,16 @@ html {
 	--wp-global-header--text-color: var(--wp--preset--color--white);
 }
 
+.has-charcoal-0-color,
+.has-charcoal-1-color,
 .has-charcoal-2-color {
 	--wp-global-header--link-color: var(--wp--preset--color--charcoal-2);
 	--wp-global-header--link-color--active: var(--wp--preset--color--deep-blueberry);
 	--wp-global-header--text-color: var(--wp--preset--color--charcoal-2);
+
+	&.has-background[style*="background-color:"] {
```

I noticed that editing the global header background color in the editor can cause the class name to change, even if the color is the same. This would make the hover color reset here ineffective. 

Do you think adding something like `&[class*="-background-color"]` makes sense?
```suggestion
	&.has-background[style*="background-color:"],
	&[class*="-background-color"] {
```